package.json

```diff
@@ -1,131 +1,137 @@
 {
-  "name": "front-end",
-  "version": "0.1.0",
-  "private": true,
```

Comment:
Whoops!  It looks like you included the invoked prettier changes on this file in the "add prettier" commit -- this makes it hard to see what changes you made to add prettier.

Sorry for this, but would it be possible to split that out?  This would involve reverting this file to the previous state, re-making the change in this commit, and then adding the prettified version to the  "invoke oprettier" commit

Reply:
Oops, I missed this. (Agree!)

Reply:
@hminsky2002 I'm not seeing this change in the most recent code.

So to be clear we're talking about git history hygiene: in 4fe57a3be50d2ab12b549fd079e7b0ffdfc7673b, it's impossible to tell which `package.json` lines are new, because the file has simultaneously been Prettified. The Prettification of the file shouldn't happen until c9b81c71974a2cf9d39bbd3bc8587469fba7daa1, which of course you will then flag as ignored in the very next commit.

Sorry to keep asking for changes, but could you edit these so that the `package.json` changes in the first commit are only what's needed to add Prettier (including the new `lint:` scripts, etc.), then reserve the file Prettification for the next commit? 🙏